resources/assets/js/components/FriendsList.vue

```diff
@@ -129,13 +129,16 @@
                     text: 'This will delete @' + friend.username + ' from your list of friends',
                     type: 'warning',
                     showCancelButton: true
-                }, function () {
+                }, () => {
                     vm.$http.delete('/api/conferences/' + vm.conferenceId + '/' + vm.key + '/' + friend.id)
-                        .then(function () {
-                            vm.list.$remove(friend)
-                        });
+                        .then(() => { vm.list.$remove(friend) });
                 });
             },
+
+            metFriend: function (friend) {
+                this.$http.patch('/api/conferences/' + this.conferenceId + '/' + this.key + '/' + friend.id, { met: ! friend.met })
```

Comment:
I'm 100% fine with either way. If every time we use it we really just want to toggle it, then let's make "toggle". If every time we use it we want to be explicit about which way we're setting it, then let's make unique method names. Whichever is clearer. :)